cmd/dep/ensure.go

```diff
@@ -304,27 +304,22 @@ func getProjectConstraint(arg string, sm gps.SourceManager) (gps.ProjectConstrai
 }
 
 func checkErrors(m map[string]pkgtree.PackageOrErr) error {
-	noGoErrors, pkgErrors := 0, 0
-	for _, poe := range m {
+	buildErrors, noGoErrors := []string{}, []string{}
```

Comment:
I tried to benchmark this and both don't allocate unless you try to append to the slice. I believe the compiler optimize these cases.

Anyhow, `var buildErrors, noGoErrors []string` is shorter. :grin: